src/components/Welcome.jsx

```diff
@@ -12,7 +12,39 @@ export class Welcome extends React.Component {
   }
 
   render() {
-    return <h1 className="app">Aurthors Haven</h1>;
+    return (
+      <fragment>
+        <nav className="navbar navbar-light navbar-expand-md navbar-fixed-top navigation-clean-button">
+          <div className="container">
+            <a href="/" className="navbar-brand">
+              <span>Authors-haven</span>
+            </a>
+            <button
+              data-toggle="collapse"
+              data-target="#navcol-1"
+              type="button"
+              className="navbar-toggler"
+            >
+              <span className="sr-only">Toggle navigation</span>
```

Comment:
@Manorlds-Eaglespark, Why did you put the navigation bar in this file.

Reply:
@Manorlds-Eaglespark I believe it would be in our best interest to remove this jsx with the `nav` because it gives us 2 navbars displayed as opposed to the expected one that has been added to our `Routes`.